sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ManipulationPreconditionsTest.java

```diff
@@ -663,4 +662,105 @@ void testMetricStatusManipulationMSRMTActivationStateFAILBadSecondManipulationFa
+        verify(mockManipulations).setComponentActivation(metricStateHandle, startActivationState);
+        verify(mockManipulations).setMetricStatus(metricStateHandle, metricCategory, activationState);
+    }
+
+    @Test
+    @DisplayName("testMetricStatusManipulationSETActivationStateNOTRDYBad: First Manipulation failed.")
+    void testMetricStatusManipulationSETActivationStateNOTRDYBadFirstManipulationFailed() {
+
+        // given
+
+        final ComponentActivation startActivationState = ComponentActivation.OFF;
+        final MetricCategory metricCategory = MetricCategory.SET;
+        final ComponentActivation activationState = ComponentActivation.NOT_RDY;
+        final String metricStateHandle = "metricStateHandle";
+
+        setupMetricStatusManipulationSET(metricCategory, metricStateHandle);
+
+        when(mockManipulations.setComponentActivation(metricStateHandle, startActivationState))
+                .thenReturn(ResponseTypes.Result.RESULT_FAIL);
+        when(mockManipulations.setMetricStatus(metricStateHandle, metricCategory, activationState))
+                .thenReturn(ResponseTypes.Result.RESULT_SUCCESS);
+
+        // when
+
+        final boolean result =
+                ManipulationPreconditions.MetricStatusManipulationSETActivationStateNOTRDY.manipulation(injector);
+
+        // then
+        assertFalse(result);
+        verify(mockManipulations).setComponentActivation(metricStateHandle, startActivationState);
+    }
+
+    @Test
+    @DisplayName("testMetricStatusManipulationSETActivationStateNOTRDYBad: Second Manipulation Failed.")
+    void testMetricStatusManipulationSETActivationStateNOTRDYBadSecondManipulationFailed() {
+
+        // given
+        final ComponentActivation startActivationState = ComponentActivation.OFF;
+        final MetricCategory metricCategory = MetricCategory.SET;
+        final ComponentActivation activationState = ComponentActivation.NOT_RDY;
+        final String metricStateHandle = "metricStateHandle";
+
+        setupMetricStatusManipulationSET(metricCategory, metricStateHandle);
+
+        when(mockManipulations.setComponentActivation(metricStateHandle, startActivationState))
+                .thenReturn(ResponseTypes.Result.RESULT_SUCCESS);
+        when(mockManipulations.setMetricStatus(metricStateHandle, metricCategory, activationState))
+                .thenReturn(ResponseTypes.Result.RESULT_FAIL);
+
+        // when
+        final boolean result =
+                ManipulationPreconditions.MetricStatusManipulationSETActivationStateNOTRDY.manipulation(injector);
+
+        // then
+        assertFalse(result);
+        verify(mockManipulations).setComponentActivation(metricStateHandle, startActivationState);
+        verify(mockManipulations).setMetricStatus(metricStateHandle, metricCategory, activationState);
+    }
+
+    private void setupMetricStatusManipulationSET(final MetricCategory metricCategory, final String metricStateHandle) {
```

Comment:
Identical to _setupMetricStatusManipulationMSRMT_ except for the name. Since the method is already parameterized, the category can be removed from the name.

Reply:
Indeed. Removed duplicated code.